src/api/service.rs

```diff
@@ -105,7 +107,7 @@ pub(crate) fn evaluate(
                 }
             }
             let policy_evaluation_metric = metrics::PolicyEvaluation {
-                policy_name,
+                policy_name: policy_id.to_string(),
```

Comment:
We have to use `to_string()` here because `policy_name` is a `String`, while here is a `PolicyID` type.

We call `to_string()` to convert `PolicyID` using the implementation of the `Display` trait (which we provided)